### PR TITLE
chore(footer): ui changes. Fixes MEMB-432

### PR DESCRIPTION
**Pull request by mibaru-aegee**

### CODE REVIEW

**Comment by mibaru-aegee** [comment]
3 rem padding top and bottom still seems pretty big, changing it to 1 rem

File: src/components/layout/FooterBar.vue [hunk 1]
```diff
@@ -2,14 +2,15 @@
   <footer class="footer">
     <div class="container">
       <div class="content has-text-centered">
-        <p>© 2017-{{ year }} AEGEE - Frontend version {{ pkg.version }}</p>
-        <p>Powered by <a href="//myaegee.atlassian.net/wiki/spaces/GENERAL">Online membership system</a></p>
         <p>
-          <router-link :to="{ name: 'oms.confluence', params: { id: 'terms-of-service' } }">Legal info (simple)</router-link> &bull;
```
Comment (serge1peshcoff) [inline src/components/layout/FooterBar.vue:8]: this one shouldn't be changed if I'm not mistaken, we should use the Confluence one (as we can change it without re-deploying)

File: src/components/layout/FooterBar.vue [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
+          <router-link :to="{ name: 'oms.legal.simple'}">Legal info</router-link> &bull;
           <!--<router-link :to="{ name: 'oms.legal.full' }">(complex)</router-link> &bull;-->
-          <router-link :to="{ name: 'oms.about' }">About MyAEGEE</router-link> &bull;
+          <!-- <router-link :to="{ name: 'oms.about' }">About MyAEGEE</router-link> &bull; -->
+          <a href="/about" target="_blank">About MyAEGEE</a> &bull;
```
Comment (serge1peshcoff) [inline src/components/layout/FooterBar.vue:9]: https://stackoverflow.com/a/49654382/1206421 this one should work

File: src/components/layout/FooterBar.vue (hunk 1 above)
Comment (serge1peshcoff) [reply src/components/layout/FooterBar.vue:8]: Actually it's broken for now, but changing the param from id to page_id should fix it I suppose

Comment (mibaru-aegee) [reply src/components/layout/FooterBar.vue:8]: I changed it, but it doesn't work on my machine. It just loads the login page

Comment (serge1peshcoff) [reply src/components/layout/FooterBar.vue:8]: with page_id?

Comment (mibaru-aegee) [reply src/components/layout/FooterBar.vue:8]: yes. It results in the link beeing `/pages/terms-of-service`, which does nothing

Comment (mibaru-aegee) [reply src/components/layout/FooterBar.vue:8]: @serge1peshcoff what should I do with the link?

Comment (serge1peshcoff) [reply src/components/layout/FooterBar.vue:8]: Let's just rename it to have page_id, I'll fix it not loading later (apparently my cors proxy doesn't work as intended, given that we now have redirect from oms-project.atlassian.net to myaegee.atlassian.net)

**Comment by serge1peshcoff** [comment]
@WikiRik I suppose we can close this?

**Comment by WikiRik** [comment]
Yes

**Comment by serge1peshcoff** [comment]
Superseded by the PR above (which is on master and prod already).

**Pull request closed without merging**